### DISCUSSION TITLE
Trouble Paginating Taxonomy

### DISCUSSION THREAD

**Discussion by rdbinteractive** [statamic/cms#5574] [Q&A]
I'm trying to paginate a taxonomy and getting the following error:
` Call to a member function all() on array `

I am following the docs from here: https://statamic.dev/tags/taxonomy#entries, specifically the part that says:
_"You're free to use filtering or sorting parameters on the entries pair that you'd find on the [collection tag](https://statamic.dev/tags/collection)."_

Here is the code I'm using:

```
Page: /articles/categories/category-name

{{ entries paginate="6" as="collection_entries" }}
    {{ if no_results }}
        {{ partial:partials/elements/four_oh_four }}
    {{ /if }}

    {{ collection_entries }}
        {{ partial:partials.blocks.archive.card }}
    {{ /collection_entries }}

    {{ partial:partials/elements/paginate }}
{{ /entries }}
```
With no parameters, `{{ entries }}` works as expected. I get all articles from the correct category.

It seems that using _any_ parameter triggers this error for me. 

I'd really appreciate it if someone could explain what I'm doing wrong.

Thank you for your time.

**Comment by edalzell** [comment]
I don't think you can paginate those as `entries` is not a tag, but data, so you're dealing w/ modifiers not parameters.

Can you show an example of another modifier that throws that error for you?
